### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by saiyan86**
finished Chinese translation for Markdown file.

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -1,114 +1,201 @@
...
-
-Fili是用来搭建和维护RESTful web服务的Java框架，主要应用于时间数据的访问和分析。Fili的[访问API](docs/end-user-api.md)基于HTTP GET，十分简洁，易于使用，大大简化了[metic和dimension定义](docs/end-user-api.md)，数据存储，以及访问查询优化。Fili是一个适用于大数据，高拓展性的框架，目前完全支持[Druid](http://druid.io)数据库，Fili有很强的扩展性，可以兼容其他任何数据库。
+<div id="table-of-contents">
+<h2>Table of Contents</h2>
+<div id="text-table-of-contents">
+<ul>
+<li><a href="#fili">1. Fili</a>
+<ul>
+<li><a href="#gitter-users-google-group-developers-google-group">1.1. 社区</a></li>
+<li><a href="#">1.2. 使用简介</a></li>
```
Comment (QubitPi) [inline README-zh.md]: `href="#"` doesn't link to the actual section

File: README-zh.md [hunk 2]
```diff
@@ -1,114 +1,201 @@
...
+<li><a href="#">1.3. 版本</a>
```
Comment (QubitPi) [inline README-zh.md]: `href="#"` doesn't link to the actual section

File: README-zh.md [hunk 3]
```diff
@@ -1,114 +1,201 @@
...
+<ul>
+<li><a href="#deprecated">1.3.1. @Deprecated</a></li>
+</ul>
+</li>
+<li><a href="#">1.4. 下载</a></li>
```
Comment (QubitPi) [inline README-zh.md]: `href="#"` doesn't link to the actual section

File: README-zh.md [hunk 4]
```diff
@@ -1,114 +1,201 @@
...
+<li><a href="#">1.5. 扩展</a></li>
```
Comment (QubitPi) [inline README-zh.md]: `href="#"` doesn't link to the actual section

File: README-zh.md [hunk 5]
```diff
@@ -1,114 +1,201 @@
...
 
-Active development happens on the patch version of the highest minor version.
+Active development happens on the patch version of the highest minor
+version.
+
+### @Deprecated<a id="deprecated" name="deprecated"></a>
+
 
-### @Deprecated
+标注了=@Deprecated=的API将在后续的发布中移除。过时的API会被维护一个发布周期，届时请尽快更新API。API不再维护之后随时会被清除。
```
Comment (QubitPi) [inline README-zh.md]: Why do we add "=" around `@Deprecated` ?

File: README-zh.md [hunk 6]
```diff
@@ -1,114 +1,201 @@
...
+
+
+<tr>
+<td class="left">高性能 slice routing</td>
+<td class="left">查询权重检查 Query weight checks</td>
+</tr>
+
+
+<tr>
+<td class="left">Dimension joins (both annotation and filtering)</td>
```
Comment (QubitPi) [inline README-zh.md]: This is rendered wrongly.

File: README-zh.md [hunk 7]
```diff
@@ -1,114 +1,201 @@
...
+<td class="left">[Rich usage metrics](monitoring-and-operations.md)</td>
```
Comment (QubitPi) [inline README-zh.md]: This is rendered wrongly

**Review by ghost — CHANGES_REQUESTED** [review]
Looks like org-mode doesn't parse to generate correct content somewhere. @garyluoex @michael-mclawhorn @cdeszaq What's your thoughts?

**Comment by michael-mclawhorn** [comment]
@QubitPi  I'm not sure I understand what you're referring to by 'org-mode'

**Comment by QubitPi** [comment]
duplicate of https://github.com/yahoo/fili/pull/444

**Pull request closed without merging**